.github/workflows/test_dropsondes.yaml

```diff
@@ -3,6 +3,7 @@ name: test
 on: [push, pull_request]
 
 env:
+  # Increase this value to reset cache
   CONDA_CACHE_NUMBER: 0
```

Comment:
I think this variable is not needed anymore since you don't use a cache for the condo environment